fog/view/connection/src/bin/client.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2018-2023 The MobileCoin Foundation
+
+use clap::Parser;
+use grpcio::EnvBuilder;
+use mc_attest_verifier::{Verifier, DEBUG_ENCLAVE};
+use mc_common::logger::{create_root_logger, log, Logger};
+use mc_fog_uri::{ConnectionUri, FogViewUri};
+use mc_fog_view_connection::FogViewGrpcClient;
+use mc_fog_view_protocol::FogViewConnection;
+use mc_util_grpc::GrpcRetryConfig;
+use std::sync::Arc;
+
+/// Parser for fog-view-client binary
+#[derive(Clone, Debug, Parser)]
+#[clap(
+    name = "fog-view-client",
+    about = "Test client for Fog view service.",
+    version
+)]
+struct Config {
+    /// Fog View service URI.
+    #[clap(long, env = "MC_FOG_VIEW")]
+    pub fog_view: FogViewUri,
+}
+
+fn main() {
+    // Logging must go to stderr to not interfere with STDOUT
+    std::env::set_var("MC_LOG_STDERR", "1");
+    let config = Config::parse();
+    let logger = create_root_logger();
+
+    let mut grpc_client = build_fog_view_conn(config.fog_view, logger.clone());
+
+    match grpc_client.request(0, 0, vec![]) {
+        Ok(resp) => {
+            log::info!(logger, "Got response:\n{:?}", resp);
```

Comment:
:upside_down_face: nit
```suggestion
            log::info!(logger, "Got response:\n{resp:?}");
```